### ISSUE TITLE
Adding SVG icon not meeting accessibility requirement 

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/OrionStatelessComponents__ods-button#42]
The current support for adding an SVG icon to a button is not properly supporting a11y standards. 

https://github.com/AlaskaAirlines/OrionStatelessComponents__ods-button/blob/92c0b2b4b225a21c0975ff5be856be5278ec24e9/src/component-base.js#L130

See https://www.w3.org/TR/WCAG20-TECHS/G201.html for spec and Rule 8 of https://www.sitepoint.com/15-rules-making-accessible-links/ for a suggested use case

**Comment by blackfalcon** [comment]
The icon issue has been addressed in the Orion Icon repo. ods-button, and the recently updates ods-hyperlink have dependencies on the Orion Icon lib. By having the latest version, this will address the issue.
